tracing/src/lib.rs

```diff
@@ -869,6 +869,14 @@ pub mod __macro_support {
     pub type Once = tracing_core::Once;
 }
 
+#[doc(hidden)]
```

Comment:
We may want to consider making this public API, but I think that should be considered separately from fixing the monomorphization issue, so let's save that for another PR.